src/utils/events/event-manager.js

```diff
@@ -76,38 +76,46 @@ export default class EventManager {
    */
   on(event, handler) {
     if (typeof event === 'string') {
-      // Special handling for gestural events.
-      const recognizerEvent = EVENT_RECOGNIZER_MAP[event];
-      if (recognizerEvent) {
-        // Enable recognizer for this event.
-        this.manager.get(recognizerEvent).set({enable: true});
-
-        // Alias to a recognized gesture as necessary.
-        const eventAlias = GESTURE_EVENT_ALIASES[event];
-        if (eventAlias && !this.aliasedEventHandlers[event]) {
+      this._onOne(event, handler);
+    } else {
+      // If `event` is a map, call `on()` for each entry.
+      for (const eventName in event) {
+        this._onOne(eventName, event[eventName]);
+      }
+    }
+  }
+
```

Comment:
I prefer to see all public method first, followed by private (underscore ) methods, it should be easy to scan a class from top to bottom to see the public interface.